libraries/abstractions/secure_sockets/test/iot_test_tcp.c

```diff
@@ -3262,7 +3262,7 @@ TEST( Full_TCP, AFQP_SOCKETS_htons_HappyCase )
 }
 /*-----------------------------------------------------------*/
 
-TEST( Full_TCP_Extended, test_dns_multiple_addresses )
+TEST( Full_TCP_Extended SOCKETS_dns_multiple_addresses )
```

Comment:
Oh, you missed a comma here :)
Causing tests to fail

Reply:
fixed